crates/cxx-qt-lib/src/core/qpointf.rs

```diff
@@ -73,6 +91,48 @@ impl fmt::Display for QPointF {
     }
 }
 
+impl std::ops::Add for QPointF {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
+        ffi::qpointf_plus(&self, &other)
+    }
+}
+
+impl std::ops::Sub for QPointF {
+    type Output = Self;
+    fn sub(self, other: Self) -> Self {
+        ffi::qpointf_minus(&self, &other)
+    }
+}
+
+impl std::ops::Mul<f32> for QPointF {
+    type Output = Self;
+    fn mul(self, rhs: f32) -> Self {
+        ffi::qpointf_mul_f32(rhs, &self)
+    }
+}
+
+impl std::ops::Mul<f64> for QPointF {
+    type Output = Self;
+    fn mul(self, rhs: f64) -> Self {
+        ffi::qpointf_mul_f64(rhs, &self)
+    }
+}
+
+impl std::ops::Mul<i32> for QPointF {
```

Comment:
QPointF only has `QPointF &QPointF::operator*=(qreal factor)` did we want i32 as well ?

Reply:
Fixed